### ISSUE TITLE
[BUG] when running `cert`, helm_deploy is running according to console output but does not show up in total tests in summary or the results yml

### ISSUE THREAD

**Issue by agentpoyo** [cncf/cnf-testsuite#1445]
**Describe the bug**
Helm_deploy not getting counted in summary total tests nor showing up in the results.yml file.

**To Reproduce**
run `cert` against a CNF, you will see `helm_deploy` run but not show up in results file where it also is not getting counted in total tests in the summary.

**Expected behavior**
helm_deploy should show in the results and be counted in total tests in summary.

**Comment by HashNuke** [comment]
Checked the test. It doesn't seem to be destructive. The test only checks for `helm_used` in the startup info ConfigMap to determine pass or fail.

* Removing the requirement for the destructive flag from the test.
* Also updating the spec tests to not pass the destructive flag.

**Comment by HashNuke** [comment]
Had to update another spec test for verbose option. That spec test was running the `helm_deploy` task as a part of the spec test.

**Comment by agentpoyo** [comment]
It's showing up now in the results.yml but not adding to the summary total:

```
RESULTS SUMMARY
  - 37 of 56 total tests passed
  - 13 of 13 essential tests passed
Results have been saved to results/cnf-testsuite-results-20220506-141656-996.yml
pair@cnfdev03:~/workspace/drew/cnf-testsuite$ less results/cnf-testsuite-results-20220506-141656-996.yml
pair@cnfdev03:~/workspace/drew/cnf-testsuite$ cat results/cnf-testsuite-results-20220506-141656-996.yml | grep helm_deploy
- name: helm_deploy
```

**Comment by HashNuke** [comment]
Confirmed with Drew. I was able to reproduce the same issue.

TLDR: Total test count is different on the test output and on the results file.

![CleanShot 2022-05-09 at 21 44 57@2x](https://user-images.githubusercontent.com/84005/167452937-0bf53729-ea7d-4460-9fc8-794e08f2f14b.png)

**Comment by HashNuke** [comment]
Added the results to a sheet to inspect.

**Observations:**
* There is 1 N/A test which is causing the test count discrepancy.
* There some tests with skipped results which could be N/A instead.

<img width="993" alt="CleanShot 2022-05-10 at 07 13 23@2x" src="https://user-images.githubusercontent.com/84005/167525828-9d27e931-dd4e-4c9c-aeee-de0566e9c142.png">

**Comment by agentpoyo** [comment]
The test being deducted from the summary total for "total passed tests" is a N/A test, so this is accurate according to the summary. Since the test is N/A, it's not a "passed" test.

**Comment by agentpoyo** [comment]
> @agentpoyo following up on #1441, does `helm_deploy` appear in the summary and results when the `cert` command is run?

Yes, helm_deploy is now in the results. The "56 tests" was getting 1 deducted because it was the "pod_dns" test as N/A test, so the summary is accurate stating X of 56 tests passed since an N/A test does not actually run nor pass.
